src/explore-education-statistics-common/src/polyfill.js

```diff
@@ -41,6 +41,26 @@ if (!Element.prototype.matches) {
     Element.prototype.webkitMatchesSelector;
 }
 
+/* eslint-disable */
```

Comment:
You shouldn't need this code AND the polyfill adding in the package.json.  The polyfill that has been added via lerna will contain this same code.  What you need to do is to add an import statement at the top of this file that imports the polyfill that you installed via lerna.  Happy to take you through it.